addons/udes_stock/models/stock_location.py

```diff
@@ -703,6 +725,15 @@ def _get_stock_drift(self, adjustments_request):
 
         return stock_drift
 
+    def _process_inventory_check_validation(self):
+        """Method will be override if advanced stock check is installed"""
+        Inventory = self.env["stock.inventory"]
+
+        inventory_data = self.get_inventory_validated_data()
+        inventory = Inventory.create(inventory_data)
+        inventory.action_start()
+        inventory.sudo().button_done()
```

Comment:
Any reason we use button_done() instead of action_done()? Button done can open a pop-up, which we probably don't want in this case.